internal/util/projutil/project_util.go

```diff
@@ -129,7 +129,7 @@ func CheckGoProjectCmd(cmd *cobra.Command) error {
 	if IsOperatorGo() {
 		return nil
 	}
-	return fmt.Errorf("'%s' can only be run for Go operators; %s does not exist", cmd.CommandPath(), mainFile)
+	return fmt.Errorf("'%s' can only be run for Go operators; %s or %s or %s does not exist", cmd.CommandPath(), goModFile, gopkgTOMLFile, pkgApiDir)
```

Comment:
When listing a list of items using a comma separate list and a single or will suffice.
```
return fmt.Errorf("'%s' can only be run for Go operators; %s, %s or %s does not exist", cmd.CommandPath(), goModFile, gopkgTOMLFile, pkgApiDir)
```
Alternatively,
```
return fmt.Errorf("'%s' can only be run for Go operators; one of %s, %s, %s does not exist", cmd.CommandPath(), goModFile, gopkgTOMLFile, pkgApiDir)
```
I'm fine with either one, depends on how someone wants to parse the sentence :)

Reply:
👍 done